world.toml

```diff
@@ -0,0 +1,18 @@
+# By default, the root cardinal directory is the location of this config file. The root
+# directory can be changed by uncommenting this line:
+# root_dir="/path/to/cardinal/"
+
+[docker_env]
```

Comment:
as per comment on the world CLI PR, I would prefer to have this header be called [cardinal] as we might want to have multiple cardinal game project in the same repo that we want to manage from a single config file.